pfcli/deployment.py

```diff
@@ -122,6 +123,17 @@
     headers=["ID", "Type", "Cloud", "VM", "GPU", "Total Usage (days, HH:MM:SS)"],
 )
 
+deployment_event_table = TableFormatter(
+    name="Deployment Event",
+    fields=[
+        "id",
+        "type",
+        "description",
+        "created_at",
+    ],
+    headers=["ID", "Type", "Description", "Timestamp"],
+)
```

Comment:
If `event` command only shows events according to one deployment,
it would be better to delete `id` field in table, and add `id` in table name.

Reply:
We also have `id` fields in metrics table. Do you think it is better to change all the other tables in this PR?

Reply:
Well, then how about making a decision later, not in this PR?